### PR TITLE
Infrastructure and dependency updates

### PR DESCRIPTION
**Pull request by nathan-weinberg**
- Pinning `package-lock-utd` to specific version
- Added `verbose` mode to `ansible-playbook` command for easier debugging
- Renamed `ansible.yml` to `playbook.yml`

### CODE REVIEW

File: devops/playbook.yml [hunk 1]
```diff
@@ -63,10 +63,17 @@
...
+    #   shell: npx browserslist@latest --update-db
+    #   args:
+    #     chdir: /home/ubuntu/regional-rail-explorer
+
     - name: build application
       shell: npm run build
       args:
         chdir: /home/ubuntu/regional-rail-explorer
+      register: build_cmd
+      failed_when: build_cmd.rc != 0
```
Comment (idreyn) [inline devops/playbook.yml:76]: It's surprising to me that this isn't the default!

**Review by idreyn — APPROVED** [review]
Still kind of a noob with regard to all of this stuff, but it looks good to me!

File: devops/playbook.yml (hunk 1 above)
Comment (mathcolo) [reply devops/playbook.yml:76]: This isn't right, 0 already means success https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html#return-rc

**Review by mathcolo — CHANGES_REQUESTED** [review]

File: devops/playbook.yml (hunk 1 above)
Comment (nathan-weinberg) [reply devops/playbook.yml:76]: whoopsie

Comment (nathan-weinberg) [reply devops/playbook.yml]: wait, but it's a `!=` @mathcolo...

**Review by mathcolo — APPROVED** [review]

**Pull request merged**